### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by m4ss1m0g** [xaviersolau/BlazorJsonLocalization#32]
First of all, thanks for a great job, I looked at the source code, and is very well written (with comments!).

I would suggest you some improvements:

1. Change the name resources separator from - (dash) to . (point) : now is <name>-fr-FR.json by I think is better to adhere to the resx standard dot-like <name>.fr-FR.json or at least provide a parameter to specify the separator.
2. Update the Doc to specify wildcards to include ALL Resources see the [Microsoft Docs](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2015/msbuild/msbuild-items?view=vs-2015&redirectedfrom=MSDN#BKMK_Wildcards) or update the code to embed resources by defaults.
``` xml
  <ItemGroup>
    <EmbeddedResource Include="Resources\**\*.json" />
  </ItemGroup>
```
3. The JSON format: I think it's will very interesting if the libs adhere to [Javascript i18n structure](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n/Locale-Specific_Message_reference) with nested/complex objects, but I think is not quite simple to implement.

Thanks.

**Comment by xaviersolau** [comment]
Thanks for your feedback!

I will update the doc with the wildcards version of the ItemGroup declaration!

Point 1:
I may be considering to add an option in the hosted resources. Unfortunately it won't be possible for EmbeddedResources since there is some constraints with dots in the resource file name (see [this](https://docs.microsoft.com/fr-fr/dotnet/core/resources/manifest-file-names) for more information).

Basically, I can load the embedded files as soon as there are more than one dots in the file name. Even playing with
LogicalName option.

Point 3:
I will investigate this.

**Comment by xaviersolau** [comment]
working on #42

**Comment by xaviersolau** [comment]
Structured Json can be used in version [2.0.0-alpha.1](https://www.nuget.org/packages/SoloX.BlazorJsonLocalization/2.0.0-alpha.1)
